src/components/AbstractSelectableComponent.js

```diff
@@ -146,6 +166,23 @@ export default class AbstractSelectableComponent extends React.Component {
     return this.renderImages(unprojectedProps);
   }
 
+  onViewStateChange({ viewState }) {
+    const {
+      updateViewInfo = () => {
+        console.warn('AbstractSelectableComponent updateViewInfo from onViewStateChange');
+      },
+    } = this.props;
+    // Update the viewport field of the `viewInfo` object
+    // to satisfy components (e.g. CellTooltip) that depend on an
+    // up-to-date viewport instance (to perform projections).
+    this.viewInfo.viewport = (new OrthographicView()).makeViewport({
```

Comment:
Looking at this in context, it's possible that only creating the `OrthographicView` once, like you had originally, would be a performance improvement... but let's wait and see if there's actually a problem.